src/main/scala/cromwell/engine/backend/BackendType.java

```diff
@@ -1,5 +1,16 @@
 package cromwell.engine.backend;
 
 public enum BackendType {
-    LOCAL, JES, SGE;
+    LOCAL {
+        @Override
+        public String displayName() {
+            return "Local";
+        }
+    },
+    JES,
+    SGE;
```

Comment:
do we need an enum? i remember when we had the initial discussion we envisioned all the instances to be dealt with equally, thus we only care of the collection of instances but not an enum of types.
I guess some design changes happened since last time I looked.
besides design styles, my main concern is on the component coupling metrics and the associated effect on cost of maintainability (coding time) of the project. (See Martin's metrics)